src/components/panel/panel.js

```diff
@@ -35,12 +35,12 @@ angular
  *   var panelRef;
  *
  *   function showPanel($event) {
- *     var panelPosition = $mdPanelPosition
```

Comment:
@ErinCoughlan Not sure that I understand this comment. I have declared both the position and animation through the $mdPanel service. See lines 38 and 43. I am probably just confused.